data/console/nginx_vhost.conf

```
@@ -0,0 +1,30 @@
+server {
```

See the following suggestion for the `nginx_vhost.conf`, which includes IPv6 on the `default_server` so that also `curl -6 http://localhost` would work.

```suggestion
server {
```
server {
  listen 80 default_server;
  listen [::]:80 default_server;

  server_name localhost;

  root /srv/www/htdocs;
  index index.html;

  location / {
    try_files $uri $uri/ =404;
  }
}

server {
  listen 80;
  listen [::]:80;
  listen 81;
  listen [::]:81;

  server_name vhost;

  return 301 https://$host$request_uri;
}

server {
  listen 443 ssl http2;
  listen [::]:443 ssl ipv6only=on http2;
  listen 444 ssl http2;
  listen [::]:444 ssl ipv6only=on http2;

  server_name vhost;

  root /srv/www/htdocs;
  index index.html;

  location / {
    try_files $uri $uri/ =404;
  }

  ssl_certificate /etc/nginx/ssl/cert.pem;
  ssl_certificate_key /etc/nginx/ssl/key.pem;
}
```

---

See http://pasta.qam.suse.de/WyszsucM for quick copy&paste